### ISSUE TITLE
IDE0007 false positive

### ISSUE THREAD

**Issue by JosefPihrt** [dotnet/roslyn#23705]
**Version Used**: VS 15.5 Community

**Steps to Reproduce**:

```csharp
class ClassName
{
    void Bar<T>(out T value)
    {
        Bar(out T value2); // IDE0007

        value = default(T);
    }
}
```

**Comment by sharwell** [comment]
Duplicate of #23250
